src/patternfly/components/Wizard/wizard.scss

```
@@ -58,6 +58,11 @@
   --#{$wizard}__nav-link-main--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
   --#{$wizard}__nav-link-main--BackgroundColor: var(--pf-t--global--background--color--action--plain--default);
   --#{$wizard}__nav-link-main--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$wizard}__nav-link-main--BorderColor: transparent;
+  --#{$wizard}__nav-link-main--BorderWidth: var(--pf-t--global--border--width--regular);
```

Since it will also support FF/Windows HCM, can you go ahead and update the width to "0" so the border doesn't show by default there? I think you'll end up with less overall code that way, too.

```suggestion
  --#{$wizard}__nav-link-main--BorderColor: var(--pf-t--global--border--color--high-contrast);
  --#{$wizard}__nav-link-main--BorderWidth: 0;
```